oio/container/backup.py

```diff
@@ -721,11 +717,14 @@ def read(size):
                         kwargs['mime_type'] = hdrs['mime_type']
                         del hdrs['mime_type']
 
-                    self.proxy.object_create(account, container,
-                                             obj_name=inf.name,
-                                             append=append,
-                                             data=read(inf.size),
-                                             **kwargs)
+                    ret = self.proxy.object_create(account, container,
```

Comment:
That method is documented as returning a `list` of chunks, size and hash of the what has been uploaded.
Why not assign a tuple is the return of the method? It would be clearer.
And if the function ever returns something different (without raising any exception), it is an error.